src/core/services/TransactionService.ts

```diff
@@ -65,7 +65,17 @@ export class TransactionServiceImpl implements TransactionService {
       // const decodedData = contractIface.decodeFunctionData(methodName, unsignedTx.data!.toString());
       // console.log({ decodedData });
```

Comment:
can we remove this commented code now?

Reply:
At first i did removed it but then i thought maybe it was useful to have it there just in case of need. It could be very possible that we will need to use it on `TransactionService` at some point